lib/JiraClient.ts

```diff
@@ -9,6 +9,25 @@ export class JiraClient {
         this.token = Buffer.from(`${user}:${token}`).toString('base64');
     }
 
+    public async createIssue(projectKey: string, issueType: any, summary: string): Promise<string> {
+        console.log(`Creating issue in project '${projectKey}' of type '${issueType.name}' with summary: ${summary}`);
```

Comment:
Let's don't log this, as #102 could in theory override the field to something else. And then, we can simplify JIRA_TASK_ISSUE to a single constant. 
```suggestion
        console.log(`Creating issue in project '${projectKey}' with summary: ${summary}`);
```